examples/small-spa/package.json

```diff
@@ -9,7 +9,6 @@
     "css-loader": "^3.4.2",
     "style-loader": "^1.1.3",
     "webpack": "git://github.com/ScriptedAlchemy/webpack.git#fixing-overrides",
```

Comment:
Seeing as its in dev-1 now, we could probably just point this repo at the official one?

Reply:
yeah agreed